summary_workflows/identification/specification/I_benchmark_specification.md

```diff
@@ -63,60 +72,71 @@ Fields:
 
 This BED file contains single-nucleotide position of poly(A) sites identified from the orthogonal 3'end-seq dataset.  
 Fields are the same as in Input 1.
+
+#### Input 3
+
+This GTF file contains reference genome annotation for the RNAseq dataset including untranslated regions.
```

Comment:
does it need to contain fields like "untranslated_region", "3UTR" in column 3? 
Or could it be the last exon in a transcript? (normally contains the 3'UTR).

Reply:
No, it just needs something that corresponds to 3'UTR. If it's easier to find the 3'UTR based on the last exon than to find annotation with UTRs, it's fine.